src/main/java/com/nike/cerberus/endpoints/authentication/AuthenticateIamPrincipal.java

```diff
@@ -66,7 +66,7 @@ public AuthenticateIamPrincipal(final AuthenticationService authenticationServic
     private ResponseInfo<IamRoleAuthResponse> authenticate(RequestInfo<IamPrincipalCredentials> request) {
         final IamPrincipalCredentials credentials = request.getContent();
 
-        log.info("{}: {}, IAM Auth Event: the IAM principal {} with ip: {} in attempting to authenticate in region {}",
+        log.debug("{}: {}, IAM Auth Event: the IAM principal {} with ip: {} in attempting to authenticate in region {}",
```

Comment:
I think this information is useful in production (as INFO level). It helps us troubleshoot customer issues.

Reply:
yeah, I need to revert that, it was quite noisy during the perf test.

Reply:
yes, revert